assets/pages/oneoff-contributions/components/oneoffContributionsPayment.jsx

```diff
@@ -125,7 +125,7 @@ function OneoffContributionsPayment(props: PropTypes, context) {
           context.store.getState,
         )}
         canOpen={formValidation(
```

Comment:
Should you not change the name of the `isFormEmpty` param in `formValidation`? It no longer means what it says

Reply:
line 88 in `oneoffInlineContributionsPayment.jsx`

Reply:
👍